content/presubmission_inquiry.md

```diff
@@ -1,29 +1,28 @@
-Dear Dr. Markel:
+Dear Dr. Markel,
 
-I am writing to submit a Ten Simple Rules article entitled “Ten Simple Rules for
```

Comment:
This is a general comment unrelated to this specific pull request, but for other Manubot manuscripts we found it easiest to keep one sentence per line without any hard line wrapping.  That makes it easier to review the changes in the GitHub diff interface and comment on lines that changed.

@Benjamin-Lee what do you think?  Should we adopt that strategy here?

Reply:
Given #57 , I think the switch to 1 sentence per line makes sense here or in a separate PR.

Reply:
Yes, it may make sense to merge this and then have separate PR that changes whitespace but not content.  That way the diff is easier to review.

Reply:
I can do that after this PR is reworked and merged.